package.json

```diff
@@ -24,20 +25,17 @@
         "abort-controller": "^3.0.0"
     },
     "peerDependencies": {
-        "grammy": "^1.3.0"
+        "grammy": "^1.20.1"
     },
     "devDependencies": {
         "@types/node": "^12.20.55",
-        "deno2node": "^1.7.1"
+        "deno2node": "^1.11.0"
     },
     "files": [
         "out/"
     ],
     "main": "out/mod.js",
     "type": "commonjs",
-    "exports": {
-        ".": "./out/mod.js"
-    },
```

Comment:
This needs to be removed in order to to use `grammy/types` on Node. Why???

Reply:
doesn't it break ESM or something if we remove this?

Reply:
Good point, we should try. Conversations with without it, though.

Reply:
I find lack of `"types"` suspicious... not sure if that's the issue tho.